### DISCUSSION TITLE
[Question]: 我使用样例代码Qwen2.5/examples/demo /web_demo.py，遇到了答非所问或者重复回答的

### DISCUSSION THREAD

**Discussion by fuweixia** [QwenLM/Qwen3#974] [Q&A]
### Has this been raised before?

- [X] I have checked [the GitHub README](https://github.com/QwenLM/Qwen2.5).
- [X] I have checked [the Qwen documentation](https://qwen.readthedocs.io) and cannot find an answer there.
- [X] I have searched [the issues](https://github.com/QwenLM/Qwen2.5/issues?q=is%3Aissue) and there is not a similar one.
- [X] I confirm that this is not a bug report, a feature request, or a badcase.

### Description

![image](https://github.com/user-attachments/assets/a1780f63-bd81-46b5-ad57-5ffdb45044aa)

**Comment by jklj077** [comment]
hi, which model did you use, did you use the base model?

**Comment by wenocy** [comment]
同样的情况下遇到了同样的问题同样的回答，用的qwen2.5-32b-base模型
